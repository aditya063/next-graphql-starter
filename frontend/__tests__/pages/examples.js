import React from 'react';
import { shallow } from 'enzyme';
import ExamplesPage from '../../pages/examples';

describe('ExamplesPage', () => {
  let mockProps;
  let wrapper;

  beforeEach(() => {
    jest.resetAllMocks();
    mockProps = {};
    wrapper = shallow(<ExamplesPage {...mockProps} />, {
      disableLifecycleMethods: true
    });
  });

  it('matches snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
