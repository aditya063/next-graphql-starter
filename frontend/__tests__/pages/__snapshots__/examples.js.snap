// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExamplesPage matches snapshot 1`] = `
<Fragment>
  <Head>
    <title>
      next-graphql-starter | Examples
    </title>
  </Head>
  <Examples />
</Fragment>
`;
